draft-ietf-tls-tls13.md

```diff
@@ -1058,6 +1058,9 @@ An implementation of TLS 1.3 that also supports TLS 1.2 might need to include
 changes to support these changes even when TLS 1.3 is not in use.  See the
 referenced sections for more details.
 
+Additionally, this document clarifies some compliance requirements in TLS 1.2,
+as described in {{protocol-invariants}}.
```

Comment:
It should probably be " 1.2; see" for style consistency with the rest of the DOC.